src/test/java/com/purbon/kafka/topology/integration/TopicManagerIT.java

```diff
@@ -70,7 +64,7 @@ public static void teardown() {
 
   @Before
   public void before() throws IOException {
-    kafkaAdminClient = AdminClient.create(config());
+    kafkaAdminClient = (AdminClient) container.getAdmin();
```

Comment:
```suggestion
    kafkaAdminClient =  container.getAdminClient();
```

Reply:
Done as part of the commit that changed the return type.